Automattic/wp-calypso
# Memberships section: Content access + connect stripe

## Changes proposed in this Pull Request

This blocks off access to http://calypso.localhost:3000/earn/memberships/ for the sites that should not have access there 

## Screenshots 

### No Stripe account connected

This should show up if user did not yet connect Stripe account.

![Zrzut ekranu 2019-05-13 o 18 28 45](https://user-images.githubusercontent.com/3775068/57638392-fc0ccc80-75ad-11e9-8c7f-11de4824debc.png)



### Free plan

This shows when user is on a free plan

![Zrzut ekranu 2019-05-13 o 14 03 53](https://user-images.githubusercontent.com/3775068/57621176-14b7bb00-758b-11e9-89e6-1ee826b2c3f9.png)

### Jetpack too old

This shows for JP sites that are too old

![Zrzut ekranu 2019-05-13 o 14 19 04](https://user-images.githubusercontent.com/3775068/57621215-2600c780-758b-11e9-9398-5db1a910e1a4.png)



## Testing instructions

Build calypso with `ENABLE_FEATURES=memberships npm run start`

For each one of these scenarios, pick a site and see a proper notice.

For a site that you can connect Stripe, go through the flow.
